examples/example-gateway/idls.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
The function that generates this file is not included in this PR. I just ran the func locally once.